ResearchKit/Common/ORKHelpers.h

```diff
@@ -34,6 +35,37 @@
 #import <ResearchKit/ORKErrors.h>
 
 
+#if ( defined(ORK_LOG_LEVEL_NONE) && ORK_LOG_LEVEL_NONE )
+#  undef ORK_LOG_LEVEL_DEBUG
+#  undef ORK_LOG_LEVEL_WARNING
+#  undef ORK_LOG_LEVEL_ERROR
+#endif
+
+#if ( !defined(ORK_LOG_LEVEL_NONE) && !defined(ORK_LOG_LEVEL_DEBUG) && !defined(ORK_LOG_LEVEL_WARNING) && !defined(ORK_LOG_LEVEL_ERROR) )
+#  define ORK_LOG_LEVEL_WARNING 1
```

Comment:
Well, a user guided by the developer could use `Xcode -> Devices` to see the *iPhone* console, where logs from shipping apps do show.